source/__tests__/integration/create-resource.test.js

```diff
@@ -10,7 +10,8 @@ describe('Create profile resource template requirements', () => {
   })
 
   afterEach(async () => {
-    await page.$eval('form[name="profileForm"]', e => e.reset())
+      page.waitFor(2000)
```

Comment:
the problem with these sorts of waits is that they are not conditional, and they slow the entire test suite down.  I wonder if we could have just made the timeout longer for page.$eval, etc. so it would go fast if it could, and wait longer if it couldn't go fast.